### PR TITLE
Add support for posting data and updated get data

### PR DESCRIPTION
**Pull request by ianmetcalf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,3 +106,25 @@ Would return the most recent 100 records from the first file alphabetically.
...
+
+#### PostDataPayload
+
+  | Param | Type | Description |
+  |---|---|---|
+  | source | string | The station that collected the data |
+  | files | Array<DataFile> | List of data files ( min length: 1, max length: 20 ) |
+  | files[].filename | string | Filename using the format `<client prefix>_<station>_<OneMin|Hourly|Meta>.dat` |
+  | files[].headers | DataFileHeaders | Optional headers for the file |
+  | files[].headers.meta | Record<string, string> | User defined meta data for the file |
```
Comment (dcavanagh) [inline README.md:122]: Are we going to accept any meta data or just TOA5?

File: README.md [hunk 2]
```diff
@@ -101,3 +106,25 @@ Would return the most recent 100 records from the first file alphabetically.
...
+  | files[].headers.columns | Array<string> | List of column names matching the data keys |
+  | files[].headers.units | Array<string> | List of units for the columns |
+  | files[].headers.processing | Array<string> | List of processing used for column data (Min, Max, Avg) |
+  | files[].records | Array<DataRecord> | List of data records for file ( max length: 100 combined across all files ) |
+  | files[].records[].timestamp | timestamp | The timestamp of the data record in UTC |
```
Comment (dcavanagh) [inline README.md:127]: Should we specify the timestamp format here?

**Review by dcavanagh — APPROVED** [review]
LGTM - Had 2 questions. see comments

File: README.md (hunk 1 above)
Comment (ianmetcalf) [reply README.md:122]: Yes, since this isn't just for loggernet data

File: README.md (hunk 2 above)
Comment (ianmetcalf) [reply README.md:127]: I'm not sure it belongs here, but we probably want to document it somewhere at some point.

**Pull request merged**